client/src/components/routes/browse/HostList.js

```diff
@@ -13,7 +13,7 @@ class HostList extends Component {
     super(props);
 
     this.state = {
-      activeHost: ""
+      activeHost: props.filteredHost || ""
```

Comment:
Tested it, and it returns this error whenever I click on a filter, even in cases where I go directly to browse instead of clicking on a cluster/office.
<img width="663" alt="Screen Shot 2019-03-31 at 3 08 31 PM" src="https://user-images.githubusercontent.com/35568696/55285906-ec06a800-53c6-11e9-9fd9-3941d2a89b3c.png">

Reply:
Probably because you weren't able to pass the `filteredHost` prop in the recursive case here:
https://github.com/compsat/ls-ganap/blob/e0223712d62323df4e2b300168fcbbc69024a34b/client/src/components/routes/browse/HostList.js#L41-L45

Reply:
Woops, my bad. Works now